test/gurka/gurka.cc

```diff
@@ -508,7 +508,9 @@ findEdge(valhalla::baldr::GraphReader& reader,
       const auto* forward_directed_edge = tile->directededge(i);
       // Now, see if the endnode for this edge is our end_node
       auto de_endnode = forward_directed_edge->endnode();
-      auto de_endnode_coordinates = tile->get_node_ll(de_endnode);
+      graph_tile_ptr reverse_tile = tile;
+      auto de_endnode_coordinates =
+          reader.GetGraphTile(de_endnode, reverse_tile)->get_node_ll(de_endnode);
```

Comment:
long standing bug here whwere we use the begin node tile to get the end node. for most tests we make small regions of tiles with short edges that stay in the tile, but i recently made a test with long long edges and so they crossed tile boundaries exposing this bug

Reply:
@chris-jpark this could be the bug we are seeing in your tests for transit? or could be totally unrelated